grc/pulsed_power_daq_power_calc_mul_ph_ff.block.yml

```diff
@@ -0,0 +1,109 @@
+id: pulsed_power_daq_power_calc_mul_ph_ff
```

Comment:
This block looks like it would not be related to the opencmw sink ... possibly should be a separate pull request ?

Reply:
This is because this was based on the [pulsed_power_daq-dev](https://github.com/fair-acc/gr-digitizers/tree/pulsed_power_daq-dev) branch but i set the `pulsed_power_daq-main` as the base branch... I'll try to do some cleanup there.